Automattic/wp-calypso
# Post-purchase: Automatically cancel migration when user exits migration

Related to #96115 
Depends on [#96103](https://github.com/Automattic/wp-calypso/pull/96103) 

## Proposed Changes
Cancel a migration (removing any pending stickers) in the following scenarios:
* Back button on `How To migrate` step
*  User selects `Import content only` on the `import-or-migrate` step. 

## Why are these changes being made?
*  We are facing multiple cases of the user's site having the pending label when they gave up on running a migration. 
The intent of this PR is to capture it and automatically cancel the migration, removing the pending label.

## Testing Instructions
* Enable the feature flag (DevTools> Console > and run:
`sessionStorage.setItem('flags,' 'automated-migration/pending-status')`
* Go to any migration flow 
* Follow all steps, choosing DIFM, until you reach the credential step
*  Use the back button to back to "How to Migrate"
* Open a new tab and go to `/sites` 
* Check your site is not marked as "Migration Pending"


## Pre-merge Checklist
